blog/sudoko_blog_structure.md

```diff
@@ -346,6 +346,36 @@ However, the CNN Combi Extra Conv model performed the best (marginally). The sur
 
 Now that we've found an architecture that works best, we wanted to see how far we could push it. If we took this same architecture and added additional nodes to the layers, how much better could performance become? In this section, we will explore just how good we can get the model by increasing its size further.
 
+We increased the number of nodes at all layers of our best-performing model, to create a model with a huge 123,416,729 learnable parameters, to see how far we could push the model. As you can see, it reached a very low loss number compared to our other models while training:
+
+![XL Model Training Loss](./images/xl_model_loss.png)
+
+Here's how it performed on our test dataset compared to the other models:
+
+| Model                        | % puzzles correct (single guess)     | % puzzles correct (iterative guesses)     | % numbers correct (iterative guesses)     |
+| ---------------------------- | ------------------------------------ | ----------------------------------------- | ----------------------------------------- |
+| MLP                          | 0.0                                  | 1.3                                       | 68.7                                      |
+| CNN stride 3                 | 24.7                                 | 35.5                                      | 83.3                                      |
+| CNN combi extra conv         | 52.2                                 | 70.6                                      | 92.6                                      |
+| **CNN combi extra conv XL**  | **65.6**                             | **96.1**                                  | **96.2**                                  |
+
+It is better again, especially when iteratively guessing the puzzles. This brought it up to an incredible 96.1% of all puzzles completed successfully. We then broke down the results by puzzle difficulty, and even on the very hardest sudoku puzzles (with 64 blanks/just 17 clues), our model was getting almost 60% of them correct:
+
+![alt text](./images/large_model_puzzles_correct.png)
+
+**TODO:** The model performed terribly on the other datasets and we don't understand why - should we talk about this or is it not very exciting since we have no insight into what's happening?
```

Comment:
We do not have much to say about it so would not be very exciting. If we have not done so already, it might be worth saying why we chose 4 million over the others - larger, greater blank range etc.

Reply:
We do mention this at the beginning, in the Data section